js/src/features/human-resouce/components/EmployeeLocationList.tsx

```diff
@@ -20,15 +18,8 @@ export const EmployeeLocationList: React.FC<Props> = ({
     {employeeLocations.map((item) => (
       <Timeline.Item
         key={item.id}
-        label={`${item.startedAt} - ${item.endedAt}`}
-        color={
-          isWithinInterval(
-            { start: parseISO(item.startedAt), end: parseISO(item.endedAt) },
-            new Date(),
-          )
-            ? "green"
-            : "blue"
-        }
+        label={`${item.startedAt} - ${!!item.endedAt ? item?.endedAt : 'till now'}`}
+        color={!item.endedAt ? "green" : "blue"}
```

Comment:
In ternary or full for of if operator in condition always use positive value.
```js
// BAD
!item.endedAt ? "green" : "blue"

// GOOD
item.endedAt ? "blue" : "green"
```

Reply:
fix